Sources/Examples.swift

```diff
@@ -1971,4 +1971,14 @@ private struct Examples {
         }
     ```
     """
+
+    let unusedPrivateDeclaration = """
+    ```diff
+    struct Foo {
+    --    fileprivate var foo = "foo"
+    --    fileprivate var baz = "baz"
+        var bar = "bar"
+    }
```

Comment:
This is the way I normally see diffs formatted, with two extra columns of indentation on the left, and the left-most column for the +/- symbols.

```suggestion
      struct Foo {
    -     fileprivate var foo = "foo"
    -     fileprivate var baz = "baz"
          var bar = "bar"
      }
```